src/Uno.Analyzers/Uno.Analyzers.csproj

```diff
@@ -2,6 +2,7 @@
 
   <PropertyGroup>
     <TargetFramework>netstandard2.0</TargetFramework>
+    <OutputPath>bin\$(Configuration)\</OutputPath>
```

Comment:
It's best to use this:
https://github.com/unoplatform/uno/blob/d813951d1e38c6e7a5d03d22be49234585f147d4/src/Uno.APISurfaceValidator/Uno.APISurfaceValidator.csproj#L6